schedule/microos/supportserver.yaml

```diff
@@ -0,0 +1,13 @@
+name: microos_supportserver
+description: >
+  Create a support server for multimachine tests
+conditional_schedule:
+  ssh_enroll:
+    SSH_ENROLL_PAIR:
+      '1':
+        - microos/ssh_enroll
```

Comment:
I would still rather put it to `jeos` folder as primary the wizard was intended only for jeos (previous brand name of Minimal-VM). Later we might reconsider to merge microos and jeos folders as they are getting closer to each other with each version